include/SFML/Graphics/Rect.hpp

```diff
@@ -164,12 +220,14 @@ template <typename T>
 template <typename T>
 [[nodiscard]] constexpr bool operator!=(const Rect<T>& left, const Rect<T>& right);
 
-#include <SFML/Graphics/Rect.inl>
-
 // Create type aliases for the most common types
 using IntRect   = Rect<int>;
 using FloatRect = Rect<float>;
 
+
+#include <SFML/Graphics/Rect.inl>
```

Comment:
Can you put this include statement back in its original location?

Reply:
Done